math/characteristic_polynomial/task.md

```diff
@@ -0,0 +1,46 @@
+## @{keyword.statement}
+
+@{lang.en}
+
+Given $N \times N$ matrix $M = \lbrace a_{ij} \rbrace$. Print $p(x)$ where
+
+$$p(x) = \det (xI - M) = \sum_{i = 0}^N p_i x^i$$
+
+and $I$ denotes the $N \times N$ identity matrix. We consider everything $\mathbb{Z}_{@{param.MOD}}$.
+
+@{lang.ja}
+
+$N \times N$ 正方行列 $M = \lbrace a_{ij} \rbrace$ が与えられます。$M$ の特性多項式 $p(x)$ を求めてください。
+
+$$p(x) = \det (xI - M) = \sum_{i = 0}^N p_i x^i$$
+
+ここで、$I$ は $N \times N$ の単位行列です。全ての要素は $\mathbb{Z}_{@{param.MOD}}$ で考えることとします。
+
+@{lang.end}
+
+## @{keyword.constraints}
+
+- $2 \leq N \leq @{param.N_MAX}$
```

Comment:
Sorry that I didn't mention when proposed, but I think we can include N = 1 (and possibly N = 0) as well.